Kickstarter-iOS/Features/BackerDashboardPages/SwiftUI/BackerDashboardTabView.swift

```diff
@@ -0,0 +1,65 @@
+import Library
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct BackerDashboardTabBarView: View {
+  @State var currentTab: BackerDashboardTab = .backed
+  @State var backedProjects: Int
+  @State var savedProjects: Int
```

Comment:
I don't love this; one of the benefits of SwiftUI is that we don't need to try to keep things in sync (as long as we have a model we're observing). I think it's fine if it's intended as a temporary thing, but in the long run, we should observe a model instead.